packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -790,3 +790,20 @@ test("Namespaced attribute with unquoted expression throws", function() {
     compile("<svg xlink:title={{title}}>content</svg>");
   }, /Namespaced attributes cannot be set as props. Perhaps you meant xlink:title="{{title}}"/);
 });
+
+module("Initial render of element modifiers");
+
+test("Element modifier", function() {
+  let manager = new TestModifierManager();
+  env.registerInternalModifier('foo', manager);
+
+  compilesTo(
+    "<div {{foo}}></div>",
+    "<div></div>"
+  );
+
+  let div = root.firstChild;
+
+  equal(manager.installedElements.length, 1);
+  equal(div, manager.installedElements[0]);
+});
```

Comment:
Actually, can ✂️ this test. It was an early bootstrapping thing. Now all the new tests are INUR in the other file. So we don't need to add more tests here.